test/jdk/tools/launcher/TestXcheckJdwpJNIWarnings.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Add the current year to copyright.

Reply:
File is new. Therefore changed year to 2022.